ethcore/src/miner/miner.rs

```diff
@@ -266,6 +267,7 @@ pub struct Miner {
 	notifiers: RwLock<Vec<Box<NotifyWork>>>,
 	gas_pricer: Mutex<GasPricer>,
 	service_transaction_action: ServiceTransactionAction,
+	increase_time: RwLock<u64>,
```

Comment:
Imho because it's so engine-specific it should be managed by the engine not miner, since engine is populating the header anyway.